File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Java/src/test/java/simulacionCesta/models/ProductTest.java
package simulacionCesta.models;

import org.junit.jupiter.api.Test;
import simulacionCesta.factories.ProductFactory;

import static org.junit.jupiter.api.Assertions.assertEquals;

class ProductTest {

    @Test
    void calculateTotalPricePerProductTest() {
        Product productTest = ProductFactory.create();
        productTest.unitaryPrice = 2;
        productTest.quantity = 2;
        assertEquals(4, productTest.calculateTotalPricePerProduct());
    }
}

File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Kotlin/src/test/kotlin/simulacionCesta/models/ProductTest.kt
package simulacionCesta.models

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionCesta.factories.ProductFactory

internal class ProductTest {
    @Test
    fun calculateTotalPricePerProductTest() {
        val productTest = ProductFactory.create()
        productTest.unitaryPrice = 2.0
        productTest.quantity = 2
        Assertions.assertEquals(4.0, productTest.calculateTotalPricePerProduct())
    }
}